src/block-kit/composition/Mrkdwn.tsx

```diff
@@ -0,0 +1,28 @@
+/** @jsx JSXSlack.h */
+import { JSXSlack } from '../../jsx'
+import { ObjectOutput } from '../../utils'
+import html from '../../html'
+import { BlockComponentProps } from '../Blocks'
+
+interface MrkdwnProps extends BlockComponentProps {
+  children: JSXSlack.Children<BlockComponentProps>
+  verbatim: boolean
```

Comment:
FYI according to [Slack API document](https://api.slack.com/reference/block-kit/composition-objects#text),   `verbatim` field is *optional*. Following this for user that wants to define JSON for Slack API exactly is good.